engine/uState.ml

```diff
@@ -458,7 +458,7 @@ type ('a, 'b) gen_universe_decl = {
   univdecl_extensible_constraints : bool (* Can new constraints be added *) }
 
 type universe_decl =
-  (lident list, Constraints.t) gen_universe_decl
+  (Level.t list, Constraints.t) gen_universe_decl
```

Comment:
IMO it was strange to have half of it using ids and half real levels (in the constraints)